root/html/editRecipe.html

```diff
@@ -65,11 +65,6 @@ <h3>Ingredients:</h3>
         </div>
         <h3 id="instruct">Instructions:</h3>
         <div class="instructions">
-          <input class="step" type="text" placeholder="Step 1" id="Step1" />
-          <input class="step" type="text" placeholder="Step 2" id="Step2" />
-          <input class="step" type="text" placeholder="Step 3" id="Step3" />
-          <input class="step" type="text" placeholder="Step 4" id="Step4" />
-          <input class="step" type="text" placeholder="Step 5" id="Step5" />
```

Comment:
This explains why we were seeing the instructions list default to 5, and increment again starting at 1 when appending. Good job on removing these, and leaving an empty div. Curious if it would be better to represent our instructions as an ordered list `<ol>` with list items, rather than `<div>`?

(edit: style/practice related suggestion, take with grain of salt)

Reply:
Yeah for sure, I think we have ol in the actual recipe page but divs in the edit page. So the steps in the edit page aren't numbered lists, the numbers come from the placeholder text in the input elements. In the actual recipe page, we can see ol being used to list out the steps. I guess this is up to formatting preferences, we could use ol in the edit page as well but we'll have to reflect that in the js when we append the elements